Makefile

```diff
@@ -263,19 +263,19 @@ $(MANIFEST_VERSION_FILE):
 	touch $(MANIFEST_VERSION_FILE)
 
 manifests/install.yaml: $(MANIFEST_VERSION_FILE) $(MANIFESTS)
-	kustomize build --load_restrictor=LoadRestrictionsNone manifests/cluster-install | sed "s/:latest/:$(MANIFESTS_VERSION)/" | ./hack/auto-gen-msg.sh > manifests/install.yaml
+	kustomize build --load_restrictor=none manifests/cluster-install | sed "s/:latest/:$(MANIFESTS_VERSION)/" | ./hack/auto-gen-msg.sh > manifests/install.yaml
```

Comment:
Is this your change? If so, why did you make it?

Regardless, we like to keep our PRs in scope, so we probably wouldn't want it in this PR.

Reply:
This is indeed my change - I saw that running `make pre-commit` would fail when using the `LoadRestrictionsNone` flag value:
```
❯ kustomize build --load_restrictor=LoadRestrictionsNone manifests/cluster-install | sed "s/:latest/:$(MANIFESTS_VERSION)/" | ./hack/auto-gen-msg.sh > manifests/install.yaml
zsh: command not found: MANIFESTS_VERSION
Error: illegal flag value --load_restrictor LoadRestrictionsNone; legal values: [rootOnly none]
```
requiring a manual run in order to properly generate the `manifests/install.yaml` file.  I've got Kustomize version 3.2.0 installed:
```
❯ kustomize version
Version: {KustomizeVersion:3.2.0 GitCommit:a3103f1e62ddb5b696daa3fd359bb6f2e8333b49 BuildDate:2019-09-18T18:31:04+01:00 GoOs:darwin GoArch:amd64}
```
Should I be using a different version when working with this project?

I learned of the `none` flag value from [this](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/FAQ.md#security-file-foo-is-not-in-or-below-bar) FAQ post in the Kustomize k8s-sig.

Totally makes sense if this is out-of-scope though - and would be happy to rescind and place in another PR.

Reply:
I'll fix this in another PR.